rel/i18n/emqx_schema.hocon

```diff
@@ -124,6 +124,22 @@ mqtt_max_qos_allowed.desc:
 mqtt_max_qos_allowed.label:
 """Max QoS"""
 
+mqtt_subscription_max_qos_rules.desc:
+"""Maximum Subscription QoS allowance rules. Useful to limit QoS level requested by the client in a `SUBSCRIBE` packet. Takes precedence over `max_qos_allowed` option, which then acts as the default when no rule applies."""
+
+mqtt_subscription_max_qos_rules.label:
+"""Max Subscription QoS Rules"""
+
+topic_qos_rule_match.desc:
+"""Expression evaluated over the given topic to decide if the rule applies.
+Follows `<topic-predicate>(topic, '<topic-argument>')` syntax where `<topic-argument>` is a string containing literal topic filter, and `<topic-predicate>` is one of:
+* `topic_equal`,
+* `topic_intersects`,
+* `topic_subset_of`."""
```

Comment:
we'll need some more description (with examples) for each predicate

Reply:
Expanded schema doc in a0db7099528b0e877be1a9ab1836963c62bb3024, please tell if you think it's not the best place for that.